image_batch_resizer/img_batch_resizer.py

```diff
@@ -56,11 +58,13 @@ def convert_tiff_folder(source_dir, quality, min_bar, max_size_kb, resize_to=Non
                             # Assuming this is the threshold size in KB
 
                             while img_quality > 20:
```

Comment:
now resizes image in /tmp folder and then saves only copies final compressed version in destination file.